[IMP] sales: update simplified pricelist settings (#11709)
odoo/documentation: content/applications/sales/sales/products_prices/prices/pricing.rst

Pricelists, discounts, and formulas
===================================

A *pricelist* is a method of dynamic pricing that applies a list of prices (or price rules) to
adjust sales prices. This adjustment can apply to specific customers, customer groups, sales orders,
time periods, etc., and is useful for creating pricing strategies and optimizing sales margins.

> Odoo *Sales* has a useful pricelist feature that can be tailored to fit any unique pricing strategy.
> Pricelists suggest certain prices, but they can always be overridden on the sales order.

Reviewer: Please use bold formatting for app names :)
```suggestion
Odoo **Sales** has a useful pricelist feature that can be tailored to fit any unique pricing
strategy. Pricelists suggest certain prices, but they can always be overridden on the sales order.
```

Pricelists configuration
========================

To enable pricelists in the Odoo **Sales** app, first navigate to :menuselection:`Sales app -->
Configuration --> Settings`. In the :guilabel:`Pricing` section, tick the checkbox next to the
:guilabel:`Pricelists` feature, and click :guilabel:`Save` to save all changes.

.. image:: pricing/pricelist-feature-setting.png
   :alt: How the pricelist feature setting looks in Odoo Sales.

After activating and saving the :guilabel:`Pricelists` feature, the :guilabel:`Settings` page

> reloads. From here, either select the :guilabel:`Pricelists` link (beneath the
> :guilabel:`Pricelists` feature on the :guilabel:`Settings` page), or navigate to

Reviewer: Nitpick suggestion to add the icon when it is part of a button :)
```suggestion
reloads. From here, either select the :icon:`oi-arrow-right` :guilabel:`Pricelists` link (beneath
the :guilabel:`Pricelists` feature on the :guilabel:`Settings` page), or navigate to
```

:menuselection:`Sales app --> Products --> Pricelists`.

Either option reveals the :guilabel:`Pricelists` page, in which pricelists can be created and/or
modified at any time.

.. image:: pricing/18-sales-pricelist-page.png

>    :alt: How the pricelists page looks in Odoo Sales.

Reviewer: This image (`18-sales-pricelist-page.png`) is too wide at 1554 px and is uncompressed, please reduce its width to be at or below 933px and please compress with pngquant

.. important::
   If there is no specific pricelist configured on a sales quotation, the :guilabel:`Default`
   pricelist is applied.

.. note::
   The :guilabel:`Selectable` column is only applicable to Odoo **eCommerce**. This option allows
   website visitors to choose a pricelist when shopping in your **eCommerce** website.

.. note::
   In Odoo 17 (and above), you are *not required* to enter a pricelist in the :guilabel:`Pricelist`
   field on a sales quotation form in order to confirm it (i.e. turn it into a sales order).

   The chatter section is available on pricelist forms, which allows you to add
   notes and communications on each pricelist page.

Creating and editing pricelists
-------------------------------

From the :guilabel:`Pricelists` page, either select the pricelist to edit, or click
:guilabel:`New` to create a new pricelist, which reveals a blank pricelist form that can be
configured in a number of different ways.

.. image:: pricing/18-sales-new-pricelist-form.png

>    :alt: How the pricelist detail form looks in Odoo Sales.

Reviewer: This image (`18-sales-new-pricelist-form.png`) is too wide at 1231 px and is uncompressed, please reduce its width to be at or below 933px and please compress with pngquant

When creating a new pricelist, start by adding a name for the pricelist at the blank field at top of
the form. Next, select which :guilabel:`Currency` should be used.

If working in a multi-company environment, select which company this pricelist should apply to
in the :guilabel:`Company` field. If this field is left blank, the pricelist is automatically
applied to all companies in the database.

If working in a multinational company, select the countries where this pricelist will apply under
the :guilabel:`Country Groups` column.

Price Rules tab
---------------

In the :guilabel:`Price Rules` tab, each line creates a new record that will implement customized
pricing to the sales order where the pricelist is applied. To create a new price rule, click on
:guilabel:`Add a line`, which opens a new pricelist rules form.

Then, select whether to apply this set of rules to a :guilabel:`Product` or
:guilabel:`Category`.

From here, there are several configuration options:

> - :guilabel:`Product`: If selected in the :guilabel:`Apply To` field, use this field to choose one
>    or more products to which this pricelist will apply.

Reviewer: Text in a list **must** align with the first character after the bullet and space. Please remove the extra space here :)
```suggestion
- :guilabel:`Product`: If selected in the :guilabel:`Apply To` field, use this field to choose one
  or more products to which this pricelist will apply.
```

- :guilabel:`Category`: Select one or more product categories to which this pricelist will apply.
- :guilabel:`Price Type`: Select whether the specialized pricing will fall under
  :guilabel:`Discount`, :guilabel:`Formula`, or :guilabel:`Fixed Price`. Depending on the price
  type, there will be additional configurations on how to apply the pricelist.

  - :guilabel:`Discount`: Enter the percentage to be discounted. A mark-up can be configured by
    using a negative value in this field.

> 

Reviewer: Please remove the empty line between the "Discount" and "Formula" bullet points
```suggestion
```

  - :guilabel:`Formula`: Calculate the pricelist rules based the following configuration:

    - :guilabel:`Discount`: Percentage discount to be applied.
    - :guilabel:`Round off to`: Numerical value to act as round-off multiple, to be applied after
      discount. The rounding method sets the price so that it is a multiple of the value in this
      field.

> .. note::
>    Rounding is applied *after* the discount and *before* the extra fee.
> 
>     - :guilabel:`Extra Fee`: Fixed amount to be added or subtracted once :guilabel:`Discount` and
>       :guilabel:`Round off to` have been applied.
> 
>   - :guilabel:`Fixed Price`: Enter fixed price for this pricelist. When applied, all product lines
>     in the quotation form will be updated to this price.

Reviewer: It appears this note should be nested under the "Round off to" bullet, please indent and fix the note admonition bullet formatting :)
```suggestion
      .. note::
         Rounding is applied *after* the discount and *before* the extra fee.

         - :guilabel:`Extra Fee`: Fixed amount to be added or subtracted once :guilabel:`Discount`
           and :guilabel:`Round off to` have been applied.
         - :guilabel:`Fixed Price`: Enter fixed price for this pricelist. When applied, all product
           lines in the quotation form will be updated to this price.
```

- :guilabel:`Min Qty`: Specify the minimum quantity of selected products for this pricelist to
  apply.
- :guilabel:`Validity Period`: Specify the start and end date during which this pricelist can be
  applied to quotations.

.. example::
   To formulate a 100% markup (or 2 times the cost of the product), with a $5 minimum margin, set
   the :guilabel:`Based on` field to :guilabel:`Cost`, the :guilabel:`Discount` to `-100`, and the
   :guilabel:`Margins` to `5`. This is often seen in retail situations.

   .. image:: pricing/formula-markup-cost-example.png

>       :alt: How it looks to formulate a markup cost with 5 dollar minimum margin in Odoo Sales.

Reviewer: This image (`formula-markup-cost-example.png`) is uncompressed, please compress with pngquant :)

Pricelists, discounts, and formulas
===================================

A *pricelist* is a method of dynamic pricing that applies a list of prices (or price rules) to
adjust sales prices. This adjustment can apply to specific customers, customer groups, sales orders,
time periods, etc., and is useful for creating pricing strategies and optimizing sales margins.

Odoo *Sales* has a useful pricelist feature that can be tailored to fit any unique pricing strategy.
Pricelists suggest certain prices, but they can always be overridden on the sales order.

Pricelists configuration
========================

To enable pricelists in the Odoo **Sales** app, first navigate to :menuselection:`Sales app -->
Configuration --> Settings`. In the :guilabel:`Pricing` section, tick the checkbox next to the

> :guilabel:`Pricelists` feature, and click :guilabel:`Save` to save all changes.

Reviewer 2: The image below is from a previous version (: it looks slightly different in saas-17.4+
![image](https://github.com/user-attachments/assets/84b41f9e-cf4e-4664-8298-cd213709ce3d)

Pricelists suggest certain prices, but they can always be overridden on the sales order.

Pricelists configuration
========================

> To enable pricelists in Odoo Sales app, first navigate to :menuselection:`Sales app -->
> Configuration --> Settings`. In the :guilabel:`Pricing` section, tick the checkbox next to the

Reviewer 3: ```suggestion
To enable pricelists in the Odoo Sales app, first navigate to :menuselection:`Sales app -->
Configuration --> Settings`. In the :guilabel:`Pricing` section, tick the checkbox next to the
```

Reviewer 3: The new standard is to have the app names bolded with double asterisks

> :guilabel:`Pricelists` feature. Click :guilabel:`Save` to save all changes.

Reviewer 3: ```suggestion
:guilabel:`Pricelists` feature, and click :guilabel:`Save` to save all changes.
```
Not necessary!

   :alt: How the pricelists page looks in Odoo Sales.

.. important::
   If there is no specific pricelist configured on a sales quotation, the :guilabel:`Default`

>    pricelist will be applied.

Reviewer 2: ```suggestion
   pricelist is applied.
```
suggestion for brevity. Is this still true?

.. note::
   The :guilabel:`Selectable` column is only applicable to Odoo **eCommerce**. This option allows
   website visitors to choose a pricelist when shopping in your **eCommerce** website.

.. note::
   In Odoo 17 (and above), you are *not required* to enter a pricelist in the :guilabel:`Pricelist`
   field on a sales quotation form in order to confirm it (i.e. turn it into a sales order).

>    The chatter section is available in Odoo 17 and above on pricelist forms, which allows you to add

Reviewer 2: since we mentioned the version earlier, it can be implied (:
```suggestion
   The chatter section is available on pricelist forms, which allows you to add
```

   notes and communications on each pricelist page.

> Editing and creating pricelists

Reviewer 2: maybe "creating and editing"?

-------------------------------

> From the :guilabel:`Pricelists` page, either select the pricelist you would like to edit, or click

Reviewer 2: ```suggestion
From the :guilabel:`Pricelists` page, either select the pricelist to edit, or click
```

:guilabel:`New` to create a new pricelist, which reveals a blank pricelist form that can be
configured in a number of different ways.

> .. image:: pricing/18-sales-new-pricelist-form.png

Reviewer 2: and remember to remove "align"!

   :align: center
   :alt: How the pricelist detail form looks in Odoo Sales.

When creating a new pricelist, start by adding a name for the pricelist at the blank field at top of
the form. Next, select which :guilabel:`Currency` should be used.

If working in a multi-company environment, select which company this pricelist should apply to
in the :guilabel:`Company` field. If this field is left blank, the pricelist is automatically
applied to all companies in the database.

If working in a multinational company, select the countries where this pricelist will apply under
the :guilabel:`Country Groups` column.

Price Rules tab
---------------

In the :guilabel:`Price Rules` tab, each line creates a new record that will implement customized
pricing to the sales order where the pricelist is applied. To create a new price rule, click on
:guilabel:`Add a line`, which opens a new pricelist rules form.

Then, select whether to apply this set of rules to a :guilabel:`Product` or
:guilabel:`Category`.

From here, there are several configuration options:

> - :guilabel:`Product`: Select one or more products to which this pricelist will apply.

Reviewer 2: ```suggestion
- :guilabel:`Product`: If selected in the :guilabel:`Apply To` field, use this field to choose one or more products to which this pricelist will apply.
```

  - :guilabel:`Formula`: Calculate the pricelist rules based the following configuration:

    - :guilabel:`Discount`: Percentage discount to be applied.
    - :guilabel:`Round off to`: Numerical value to act as round-off multiple, to be applied after
      discount. The rounding method sets the price so that it is a multiple of the value in this
      field.

.. note::
   Rounding is applied *after* the discount and *before* the extra fee.

    - :guilabel:`Extra Fee`: Fixed amount to be added or subtracted once :guilabel:`Discount` and
      :guilabel:`Round off to` have been applied.

  - :guilabel:`Fixed Price`: Enter fixed price for this pricelist. When applied, all product lines
    in the quotation form will be updated to this price.

- :guilabel:`Min Qty`: Specify the minimum quantity of selected products for this pricelist to
  apply.
- :guilabel:`Validity Period`: Specify the start and end date during which this pricelist can be
  applied to quotations.

.. example::
   To formulate a 100% markup (or 2 times the cost of the product), with a $5 minimum margin, set
   the :guilabel:`Based on` field to :guilabel:`Cost`, the :guilabel:`Discount` to `-100`, and the

>    :guilabel:`Margins` to `5`. This is often seen in retail situations.

Reviewer 2: good example!

>    .. image:: pricing/formula-markup-cost-example.png

Reviewer 2: Also which version did u take this screenshot from? mine looks a little different, and i was testing in a 17.4 runbot
![image](https://github.com/user-attachments/assets/b602a7db-1e31-4b83-9ba6-3ba75e15d5fe)

>       :align: center

Reviewer 2: ```suggestion
```

      :alt: How it looks to formulate a markup cost with 5 dollar minimum margin in Odoo Sales.

.. example::
   To apply 20% discounts, with prices rounded up to 9.99, set the :guilabel:`Based on` field to
   :guilabel:`Sales Price`, the :guilabel:`Discount` field to `20`, the :guilabel:`Extra Fee` field
   to `-0.01`, and the :guilabel:`Rounding Method` field to `10`.

   .. image:: pricing/formula-discount-example.png

>       :align: center

Reviewer 2: ```suggestion
```
